conda_smithy/configure_feedstock.py

```diff
@@ -1531,6 +1532,16 @@ def _load_forge_config(forge_dir, exclusive_config_file):
             if "name" in config["azure"][f"settings_{plat}"]["pool"]:
                 del config["azure"][f"settings_{plat}"]["pool"]["vmImage"]
 
+    if config["conda_forge_output_validation"]:
+        config["secrets"] = list(
+            set(
```

Comment:
Using a set would make the rerendering non-deterministic right?

Reply:
Yeah, rather than list, it should be `sorted` on ther line above